spec/lib/github_adapter_spec.rb

```diff
@@ -25,17 +39,32 @@
     end
   end
 
-  describe "#create_team" do
-    it "sends a post to the api with name/description parameters" do
-      name = "Project name"
-      desc = "Project description"
+  describe "#create_org_repo" do
+    it "calls POST with proper parameters" do
+      github_url = "https://api.github.com/orgs/chatto-hub-test2/repos"
+      expect(RestClient::Resource).to receive(:new).with(github_url, "example_user", "example_password").
+        and_return(double(post: {}.to_json))
+      subject.create_org_repo
+    end
+  end
 
-      expect_any_instance_of(RestClient::Resource).to receive(:post).with(
-        { name: name, description: desc }.to_json,
-        content_type: "application/json"
-      ).and_return({ ok: true }.to_json)
+  describe "#fork_repo" do
+    before do
+      ENV['CHATTO_HUB_ADMIN_GIT_FORK_SH'] = "fork_sh"
+      ENV['CHATTO_HUB_ADMIN_GIT_USER'] = "git_user"
+      ENV['ROOT_REPO_URI'] = "root_repo"
+    end
 
-      subject.create_team(name, desc)
+    it "executes sh script" do
+      expect(subject).to receive(:system).with(
+        'sh',
+        'fork_sh',
+        'git_user',
+        'root_repo',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.